botfront/imports/ui/components/stories/StoriesEditor.jsx

```diff
@@ -179,18 +179,7 @@ function StoriesEditor(props) {
         <>
             {editors}
             <Container textAlign='center'>
-                <Popup
-                    trigger={(
-                        <Icon
-                            name='add'
-                            link
-                            onClick={onAddNewStory}
-                            size='large'
-                            data-cy='add-story'
-                        />
-                    )}
-                    content='Add a story'
-                />
+                <Button icon='add' basic name='add' onClick={onAddNewStory} size='medium' data-cy='add-story' color='black' content='Add a story' />
```

Comment:
Why did you remove the popup ?

Reply:
Because now the button has the same text, as the popup so it will be redundant